client/signup/steps/site-or-domain/style.scss

```diff
@@ -45,6 +49,10 @@
 
 .site-or-domain__choice-text {
 	color: $gray-dark;
+
+	p {
+		min-height: 105px;
```

Comment:
This rule along with https://github.com/Automattic/wp-calypso/issues/16319, causes this ugly result:
<img width="999" alt="screen shot 2017-07-26 at 16 23 56" src="https://user-images.githubusercontent.com/3392497/28626350-0a5993d4-721f-11e7-8a2a-00f23191836c.png">

This is on newest Chromium and Safari on OSX, so I'd not consider it an edge case.